### ISSUE TITLE
Page.get_textpage_ocr() bug in rebased fitz_new version

### ISSUE THREAD

**Issue by ellacroix** [pymupdf/PyMuPDF#2691]
## Describe the bug (mandatory)
The function Page.get_textpage_ocr() doesn't work in the new rebased fitz_new version.
It raises this exception: 
``` AttributeError: module 'fitz_new.mupdf' has no attribute 'PdfocrOptions' ```

## To Reproduce (mandatory)
To reproduce, you need a Python environment with Tesseract installed.
```
import os
# os.environ['TESSDATA_PREFIX'] = 'C:\\work\\apps\\Tesseract-OCR\\tessdata'
import fitz_new as fitz
# import fitz

doc = fitz.Document('notice_8676.pdf')
for page in doc:
	TextPage = page.get_textpage_ocr(dpi=300, full=True, language='fra')
	print(TextPage.extractText())
```
[notice_8676.pdf](https://github.com/pymupdf/PyMuPDF/files/12729049/notice_8676.pdf)
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[8], line 11
      9 doc = fitz.Document('notice_8676.pdf')
     10 for page in doc:
---> 11 	TextPage = page.get_textpage_ocr(dpi=300, full=True, language='fra')
     12 	print(TextPage.extractText())

File [c:\work\apps\python_env\main\Lib\site-packages\fitz_new\utils.py:637](file:///C:/work/apps/python_env/main/Lib/site-packages/fitz_new/utils.py:637), in get_textpage_ocr(page, flags, language, dpi, full, tessdata)
    635 # if OCR for the full page, OCR its pixmap @ desired dpi
    636 if full is True:
--> 637     return full_ocr(page, dpi, language, flags)
    639 # For partial OCR, make a normal textpage, then extend it with text that
    640 # is OCRed from each image.
    641 # Because of this, we need the images flag bit set ON.
    642 tpage = page.get_textpage(flags=flags)

File [c:\work\apps\python_env\main\Lib\site-packages\fitz_new\utils.py:620](file:///C:/work/apps/python_env/main/Lib/site-packages/fitz_new/utils.py:620), in get_textpage_ocr..full_ocr(page, dpi, language, flags)
    616 mat = fitz.Matrix(zoom, zoom)
    617 pix = page.get_pixmap(matrix=mat)
    618 ocr_pdf = fitz.Document(
    619         "pdf",
--> 620         pix.pdfocr_tobytes(
    621             compress=False,
    622             language=language,
    623             tessdata=tessdata,
    624             ),
    625         )
    626 ocr_page = ocr_pdf.load_page(0)
    627 unzoom = page.rect.width / ocr_page.rect.width

File [c:\work\apps\python_env\main\Lib\site-packages\fitz_new\__init__.py:9615](file:///C:/work/apps/python_env/main/Lib/site-packages/fitz_new/__init__.py:9615), in Pixmap.pdfocr_tobytes(self, compress, language, tessdata)
   9613 from io import BytesIO
   9614 bio = BytesIO()
-> 9615 self.pdfocr_save(bio, compress=compress, language=language, tessdata=tessdata)
   9616 return bio.getvalue()
[notice_8676.pdf](https://github.com/pymupdf/PyMuPDF/files/12729010/notice_8676.pdf)


File [c:\work\apps\python_env\main\Lib\site-packages\fitz_new\__init__.py:9584](file:///C:/work/apps/python_env/main/Lib/site-packages/fitz_new/__init__.py:9584), in Pixmap.pdfocr_save(self, filename, compress, language, tessdata)
   9582 if not TESSDATA_PREFIX and not tessdata:
   9583     raise RuntimeError('No OCR support: TESSDATA_PREFIX not set')
-> 9584 opts = mupdf.PdfocrOptions()
   9585 opts.compress = compress;
   9586 if language:

AttributeError: module 'fitz_new.mupdf' has no attribute 'PdfocrOptions'
```

## Expected behavior (optional)
With the old fitz version, it should OCR the document page by page and print its text output.

## Your configuration (mandatory)
 - Operating system, potentially version and bitness: **win32**
 - Python version, bitness: **3.11.3 (tags/v3.11.3:f3909b8, Apr  4 2023, 23:49:59) [MSC v.1934 64 bit (AMD64)]**
 - PyMuPDF version, installation method (**wheel** or **generated** from source): **1.23.3, installed via pip**

**Comment by julian-smith-artifex-com** [comment]
Thanks for trying out `fitz_new` and for the detailed bug report.

It's fixed in our latest code by commit https://github.com/pymupdf/PyMuPDF/commit/6bd3a972ec0. We'll look into making a release with this fix soon.

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.23.4.
